cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:3000",
```

Comment:
small thing, but if we make client side run on :3001. we can avoid issues with running backend simultaneously.

Reply:
A valid point! Will address now